src/index.ts

```diff
@@ -1,4 +1,18 @@
 import { Deck2gisLayer } from './deckgl2gisLayer';
 import { initDeck2gisProps } from './utils';
 
+if (typeof window !== 'undefined') {
+    if ('mapgl' in window) {
+        (mapgl as any).Deck2gisLayer = Deck2gisLayer;
+    } else {
+        // Если так вышло, что плагин инициализирован раньше mapgl, поместим его во временную переменную
+        // Из нее уже сам mapgl все положит в себя.
+        if (!(window as any).__mapglPlugins) {
```

Comment:
I think we have to think about this a little bit more. Now we have a situation when the main project is decoupled from knowledge about existence of new plugin registry. What if someone need to create a new plugin and one doesn't check this plugin. There is a possibility that we end up with one more plugin registry under different name :) I think that the initialization of plugin registry should be placed inside the main project itself.

Reply:
Maybe even not in a globla scope but on the namespace of the map

Reply:
Discussed in the private messages, it is all correct :ok_hand: